.github/CODEOWNERS

```diff
@@ -55,6 +55,11 @@
 # ServiceLabel: %area-BestPractices
 # ServiceOwners:                       @g2vinay @conniey
 
+# PRLabel: %area-CloudArchitect
```

Comment:
Be sure to create this github issue label

Reply:
It might be case sensitive.